### ISSUE TITLE
Log Debug output Timestamp PR252

### ISSUE THREAD

**Issue by Kolbi** [audiconnect/audi_connect_ha#256]
Bug introduced by PR: https://github.com/arjenvrh/audi_connect_ha/pull/252

The output in the log for Timestamp looks for example:
`'carCapturedTimestamp': datetime.datetime(2024, 3, 5, 13, 35, 51, tzinfo=datetime.timezone.utc),`

This cases issues if you try to verify the json with tools like https://jsonformatter.curiousconcept.com/

**Comment by cdnninja** [comment]
I'm confused.   This is just raw response from the api.  If you disable debug this output with disappear from logs.

**Comment by t0bias-r** [comment]
@Kolbi  I agree. The timestamp is not a valid Json item, but this is how the Audi servers send them.

**Comment by Kolbi** [comment]
Funny. :) Normally Audi should fix this on their side, 

I'm thinking if it would be helpful for us to convert timestamps before LOGGER.debug...

https://github.com/arjenvrh/audi_connect_ha/blob/8065ff79c462e7daead19338a12f54b08b45f430/custom_components/audiconnect/audi_services.py#L198

**Comment by coreywillwhat** [comment]
I think its good to have the raw values at the `debug` level. It may be useful to have them at the `info` level in a more cleaned up data structure for easy reading.

Logs could use a lot more cleaning up/additions for better insight into whats happening though.

**Comment by coreywillwhat** [comment]
Also, I couldn't agree more @Kolbi I'm losing my mind removing or quoting the datetime.datetime(xxx) data so i can see it in a readable format.
